.gitignore

```diff
@@ -11,3 +11,4 @@ Thumbs.db
 /.jekyll-cache/
 Gemfile.lock
 yarn.lock
+.idea
```

Comment:
What is this used for? :)

Reply:
Jetbrains' IDEs use it for config files. It is user-specific